networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc

```diff
@@ -6,26 +6,33 @@ include::_attributes/common-attributes.adoc[]
 // Unique attribute for the version of the release notes.
 
 toc::[]
+ 
+As a cluster administrator, you can migrate to the OVN-Kubernetes network plugin from the OpenShift software-defined networking (SDN) plugin. 
 
-As a cluster administrator, you can migrate to the OVN-Kubernetes network plugin from the OpenShift SDN network plugin. The following options are available for migration:
+[WARNING]
+====
+If your cluster uses the OpenShift SDN network plugin, you cannot upgrade your cluster to {product-title} 4.17. You must initially migrate your cluster to the OVN-Kubernetes plugin and then upgrade your cluster to {product-title} 4.17.
+
+Additionally, if your cluster uses a third-party container network interface (CNI) plugin, you cannot migrate your cluster to the OVN-Kubernetes network plugin. For a complete list of these CNI plugins, see link:https://access.redhat.com/articles/5436171[Certified OpenShift CNI Plug-ins] (Red{nbsp}Hat Knowledgebase).
+====
+
+The following methods exist for migrating from the OpenShift SDN network plugin to the OVN-Kubernetes plugin:
```

Comment:
We may need to update this statement: "This is an automated process that migrates your cluster from OpenShift SDN to OVN-Kubernetes without service interruption.". AFAIK, for Egress IP 
The feature doesn’t work during live migration. However, the configuration of egress IP of OpenShiftSDN can be automatically migrated to OVN-Kubernetes once the migration is complete. That may be one reason why we call it limited live migration.